src/site/stages/build/process-cms-exports/index.js

```diff
@@ -52,19 +52,23 @@ const entityAssemblerFactory = contentDir => {
    * @return {Object} - The entity with all the references filled in with
    *                    the body of the referenced entities.
    */
-  const assembleEntityTree = (entity, parents = []) => {
+  const assembleEntityTree = (entity, ancestors = []) => {
+    // TODO: Make ancestors an array of { id: toId(), entity }
```

Comment:
It looks like this `TODO` is done on line 117, right?

Reply:
Ah, right. This was a note to self before we hopped onto a call and I forgot to clean it up. :grimacing: